contributor-book/src/getting-started/setting-up-the-environment.md

```diff
@@ -0,0 +1,29 @@
+# setting up the environment
+
+There are a couple of tools that need to be installed, and commands to be familiar with, depending on what part of the project you plan on contributing to. This section should be up to date with current project practices (as of 2024-01-26)
+
+## General
+
+There are a few commands you want to run prior to any commit for a non-draft PR:
+
+1. `cargo clippy --fix --allow-dirty`, this will run clippy and fix any issues it can, the allow dirty flag is required whenever you have uncommitted changes
+2. `cargo fmt --all`, this will run rustfmt on all files in the project
+3. `./run_checks.sh all`, this is a script located in the project root that builds and tests the project. It is required that this pass prior to merging a PR. Fair warning, running these tests can take a while[^2].
+
+## Updating the burn semver version
+
+To bump for the next version, edit the semantic version number in `burn/Cargo.toml`, and then run `cargo update` to update the lock file.
+
+
+## Contributing to the Burn (Contributor) Book
```

Comment:
I would remove `Burn`  and just use `the Contributor Book` everywhere. We can reserve `the Burn Book` for the user book as we will have way more users than contributors.

Reply:
> I would remove Burn and just use the Contributor Book everywhere. We can reserve the Burn Book for the user book as we will have way more users than contributors.

In this case I meant either contributing to the burn book, or the contributor book, the commands and tools used would be the same for both, I'll change the heading title to make that more explicit